packages/nux/src/components/dot-tip/index.js

```diff
@@ -33,7 +34,7 @@ export function DotTip( {
 	return (
 		<Popover
 			className="nux-dot-tip"
-			position="middle right"
+			position={ position || 'middle right' }
```

Comment:
You could do this with a default argument if you want to make it more explicit that the argument is optional.

```js
export function DotTip( {
	position = 'middle right',
	...
```

Totally up to you, though!

Reply:
> You could do this with a default argument if you want to make it more explicit that the argument is optional.

I'd agree this would probably be the best way to communicate it as an optional prop, and it may help future maintainability if there are changes to the component which reference `position` in more than this one location.

Reply:
@aduth Should I make this change then?

Reply:
> @aduth Should I make this change then?

It would be a worthwhile improvement if you're willing, yes.

Reply:
@aduth Made the changes. :)